doc/languages-frameworks/rust.section.md

```diff
@@ -164,6 +166,9 @@ rustPlatform.buildRustPackage {
 Please note that the code will be compiled twice here: once in `release` mode
 for the `buildPhase`, and again in `debug` mode for the `checkPhase`.
 
+Test flags, e.g., `--features xxx/yyy`, can be passed to `cargo test` via the
+`cargoTestFlags` attribute.
+
```

Comment:
> i tried it, doesn't compile, since the checkFlags are behind the double dash

the effect of checkFlags could also be explained here

Reply:
is this sufficient?